src/main/java/org/broadinstitute/hellbender/utils/Utils.java

```diff
@@ -1099,6 +1101,10 @@ protected T computeNext() {
         return stream(() -> iterator);
     }
 
+    public static Stream<VariantContext> streamVcf(final File vcf) {
+        return StreamSupport.stream(new FeatureDataSource<VariantContext>(vcf).spliterator(), false);
```

Comment:
this needs to set the stream `onClose` method.  Otherwise it will leave dangling open file handles.

Reply:
done